templates/security/notices/usn.html

```diff
@@ -2,125 +2,504 @@
+                              <span class="p-chip__value">FIPS</span>
+                              <span class="p-tooltip__message cve-chip u-hide"
+                                    role="tooltip"
+                                    id="fips-tooltip"><a href='/security/fips'>FIPS-140</a> certified package. Available with Ubuntu Pro.</span>
+                              </button>
+
+                              <button class="js-tooltip p-chip is-dense u-no-margin--bottom p-tooltip--{% if loop.first and outer_loop.first %}btm{% else %}top{% endif %}-center"
+                                    style="vertical-align: unset;"
+                                    aria-describedby="fips-updates-tooltip">
+                              <span class="p-chip__value">FIPS Updates</span>
+                              <span class="p-tooltip__message cve-chip u-hide"
+                                    role="tooltip"
+                                    id="fips-updates-tooltip"><a href='/security/fips'>FIPS-140</a> certified package with security fixes. Available with Ubuntu Pro.</span>
+                              </button>
+                            </td>
+                          </tr>
+                        {% endfor %}
+                      {% endfor %}
+                    </tbody>
+                  </table>
+                </div>
+              </div>
+            {% endif %}
+            <div class="row">
+              <div class="p-section">
+                <hr class="p-rule" />
+                <h3>Reduce your security exposure</h3>
+                <p>
+                  Ubuntu Pro provides ten-year security coverage to 25,000+ packages in Main and Universe repositories, and it is free for up to five machines.
+                </p>
+                <div class="p-cta-block">
+                  <a href="/pro" class="p-button--positive">Get Ubuntu Pro</a>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="p-section">
+            {% if notice.total_cves %}
+              <div class="u-fixed-width">
+                <h2 class="section-heading" id="references" style="scroll-margin-top: 4rem">References</h2>
+              </div>
+              {% if notice.total_cves > 29 %}
+                <form action="/security/notices/{{ notice.id }}#references" method="GET" class="p-search-box js-cve-form">
+                  {% for key, value in request.args.items() %}
+                    {% if key == "usn" %}<input type="hidden" name="usn" value="{{ value }}">{% endif %}
+                  {% endfor %}
+                  <label class="u-off-screen" for="search">Search</label>
+                  <input type="search" 
+                         id="cve" 
+                         class="p-search-box__input" 
+                         name="cve" 
+                         placeholder="Search by CVE ID" 
+                         autocomplete="on" />
```

Comment:
I'm not sure this is needed, it doesn't seem to have any effect
```suggestion
/>
```